cmd/minikube/cmd/delete_test.go

```diff
@@ -220,3 +222,60 @@ func TestDeleteAllProfiles(t *testing.T) {
 
 	viper.Set(config.ProfileName, "")
 }
+
+// TestTryKillOne spawns a go child process that waits to be SIGKILLed,
+// then tries to execute the tryKillOne function on it;
+// if after tryKillOne the process still exists, we consider it a failure
+func TestTryKillOne(t *testing.T) {
+
+	var waitForSig = []byte(`
+package main
+
+import (
+	"os"
+	"os/signal"
+	"syscall"
+)
+
+// This is used to unittest functions that send termination
+// signals to processes, in a cross-platform way.
+func main() {
+	ch := make(chan os.Signal, 1)
+	done := make(chan struct{})
+	defer close(ch)
+
+	signal.Notify(ch, syscall.SIGHUP)
+	defer signal.Stop(ch)
+
+	go func() {
+		<-ch
+		close(done)
+	}()
+
+	<-done
+}
+`)
+	td := t.TempDir()
+	tmpfile := filepath.Join(td, "waitForSig.go")
+
+	if err := os.WriteFile(tmpfile, waitForSig, 0o700); err != nil {
+		t.Fatalf("copying source to %s", tmpfile)
+	}
+
+	dontkillmeProc := exec.Command("go", "run", tmpfile)
```

Comment:
```suggestion
	processToKill := exec.Command("go", "run", tmpfile)
```
Variable name doesn't follow camel casing, also the variable name is confusing